dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -3140,19 +3140,20 @@ public ProgramOutput stopServer(boolean postStopServerArchive, boolean forceStop
 
     public ProgramOutput stopServer(boolean postStopServerArchive, boolean forceStop, boolean skipArchives,
                                     List<String> failuresRegExps, String... ignoredFailuresRegExps) throws Exception {
-        return stopServer(!IGNORE_STOPPED, postStopServerArchive, forceStop, skipArchives,
+        return stopServer(!IGNORE_STOPPED, postStopServerArchive, forceStop, skipArchives, SKIP_FEATURE_CHECK,
                           failuresRegExps, ignoredFailuresRegExps);
     }
 
     public ProgramOutput stopServerAlways(String... ignoredFailures) throws Exception {
-        return stopServer(IGNORE_STOPPED, POST_ARCHIVES, FORCE_STOP, SKIP_ARCHIVES,
+        return stopServer(IGNORE_STOPPED, POST_ARCHIVES, FORCE_STOP, SKIP_ARCHIVES, SKIP_FEATURE_CHECK,
                           Collections.emptyList(), ignoredFailures);
     }
 
     public static final boolean IGNORE_STOPPED = true;
     public static final boolean FORCE_STOP = true;
     public static final boolean POST_ARCHIVES = true;
     public static final boolean SKIP_ARCHIVES = true;
+    public static final boolean SKIP_FEATURE_CHECK = false;
```

Comment:
It makes more sense that this be true instead of false so that people can reference in their tests to do the skip like is done in the other ones.  And then you do `!` of it in the above stopServer calls.

Reply:
Done